torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -342,8 +342,18 @@ void IRPrinter::handle(const kir::NamedScalar* i) {
   os << i->name();
 }
 
-void IRPrinter::handle(const kir::IterDomain*) {
-  TORCH_INTERNAL_ASSERT(false, "Unreachable");
+void IRPrinter::handle(const kir::IterDomain* id) {
```

Comment:
where is this needed? I hit this a few times while working on the refactoring, but every time it indicated that something else needed to be updated

Reply:
Debugging matching between the IterDomain of kir and fusion was pretty challenging when I couldn't print what they were. No remaining code relies on it, but I rely on it for debugging. I had to update Type.cpp

Reply:
I think we should now be able to fill the rest out marked as Unreachable